bmu/Src/batteries.c

```diff
@@ -401,7 +401,7 @@ void HVMeasureTask(void *pvParamaters)
             CurrentBusHV = IBus;
             VoltageBusHV = VBus;
             sendCAN_BMU_stateBusHV();
-            vTaskDelay(2); // Added to prevent CAN mailbox full
+            vTaskDelay(4); // Added to prevent CAN mailbox full
```

Comment:
I don't think adding a delay is a permeant solution. We should really just have 2 loops that publish this on their own timers so that they arent beside eachother. Bc if you do this you are stalling the task from running almost a whole cycle if its at 5ms now